boot/freeldr/freeldr/arch/i386/entry.S

```diff
@@ -73,27 +74,25 @@ _FrldrStartup:
     call _InitIdt
 
 #ifndef _USE_ML
-    /* Clean out bss */
+    /* Clean out BSS -- Only for GCC build since MSVC one has it merged with TEXT (see CMakeLists.txt) */
     xor eax, eax
     mov edi, offset __bss_start__
     mov ecx, offset __bss_end__ + 3
     sub ecx, edi
     shr ecx, 2
     rep stosd
+#endif
 
     /* Pass the command line to BootMain */
     mov eax, offset cmdline
-#else
-    xor eax, eax
-#endif
 
     /* GO! */
     push eax
     call _BootMain
 
     /* We should never get here */
 stop:
-    jmp stop
+    jmp short stop
```

Comment:
Aren't jumps "short" by default?

Reply:
Yes indeed in that case it's assembled as "jmp short" even if "short" is not explicitly specified.